### PR TITLE
The final solution of bad taste in midis question

### PR DESCRIPTION
**Pull request by Razharas**
The final solution of bad taste in midis question
Added special client verb that kills all sounds currently playing for
this client
This is better because toggle midis only pauses them, not actually stops
them
Fixes https://github.com/tgstation/-tg-station/issues/11795

### CODE REVIEW

File: code/modules/admin/verbs/playsound.dm [hunk 1]
```diff
@@ -60,6 +60,6 @@ var/sound/admin_sound
 	for(var/mob/M in player_list)
 		if(M.client)
 			M << sound(null)
-	feedback_add_details("admin_verb","SS") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+	feedback_add_details("admin_verb","SAPS") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
```
Comment (RemieRichards) [inline code/modules/admin/verbs/playsound.dm]: YOU'RE RUINING THE FEEDBACK DATA NOOOO

File: code/modules/client/preferences_toggles.dm [hunk 2]
```diff
@@ -156,6 +156,13 @@
 			admin_sound.status ^= SOUND_PAUSED
 	feedback_add_details("admin_verb","TMidi") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/verb/stop_client_sounds()
+	set name = "Stop Sounds"
+	set category = "Preferences"
+	set desc = "Kills all currently playing sounds, use if admin taste in midis a shite"
+	src << sound(null)
+	feedback_add_details("admin_verb","SS") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
```
Comment (RemieRichards) [inline code/modules/client/preferences_toggles.dm]: NOOOOOO

**Comment by Razharas** [comment]
The dissent will be silenced
This will happen NO MATTER WHAT

**Comment by freerealestate** [comment]
10/1

**Comment by ike709** [comment]
9/11

**Comment by PKPenguin321** [comment]
:+1: took you fuckers long enough, now i can stop the anime midis but still listen to the shitty dubstep ones

**Comment by SconesC** [comment]
good good

File: code/modules/client/preferences_toggles.dm [hunk 3]
```diff
@@ -156,6 +156,13 @@
 			admin_sound.status ^= SOUND_PAUSED
 	feedback_add_details("admin_verb","TMidi") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/verb/stop_client_sounds()
+	set name = "Stop Sounds"
+	set category = "Preferences"
+	set desc = "Kills all currently playing sounds, use if admin taste in midis a shite"
+	src << sound(null)
+	feedback_add_details("admin_verb","SAPS") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
```
Comment (Iamgoofball) [inline code/modules/client/preferences_toggles.dm]: uhhh

**Comment by Iamgoofball** [comment]
@PKPenguin321 
>dubstep midis

File: code/modules/client/preferences_toggles.dm (hunk 3 above)
Comment (RemieRichards) [inline code/modules/client/preferences_toggles.dm]: it's done for every button, shutup

**Comment by PKPenguin321** [comment]
>**shitty** dubstep midis

File: code/modules/client/preferences_toggles.dm (hunk 3 above)
Comment (Razharas) [inline code/modules/client/preferences_toggles.dm]: They are also all copypaste and all in the "admin_verb" section for some reason despite having nothing to do with adminverbs

**Comment by Chocobro** [comment]
RIP ace of bass midi played 16 times over itself

**Pull request merged**